### PR TITLE
feat: docker compose config

### PR DESCRIPTION
**Pull request by doichev-kostia**
# Proposed changes

I've added a docker-compose config for postgres DB

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,4 +32,5 @@ build/
 ### VS Code ###
 .vscode/
 
-/src/main/resources/secrets.yaml
\ No newline at end of file
+/src/main/resources/secrets.yaml
```
Comment (Aleh1s) [inline .gitignore:35]: Ok!

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: '3.8'
+services:
+  postgres:
+    container_name: taskmate-postgres
```
Comment (Aleh1s) [inline docker-compose.yml]:
```yaml
container_name: taskmate
```

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    image: postgres:15
+    ports:
+      - '5432:5432'
+    volumes:
+      - ./db_data:/var/lib/postgresql
+    env_file:
```
Comment (Aleh1s) [inline docker-compose.yml:10]: Rename

**Review by Aleh1s — CHANGES_REQUESTED** [review]
Fix

File: docker-compose.yml (hunk 3 above)
Comment (doichev-kostia) [reply docker-compose.yml:10]: No)

**Review by Aleh1s — APPROVED** [review]
Approved!

**Pull request merged**